src/main/java/org/example/P07_LeapYear.java

```diff
@@ -1,5 +1,7 @@
 package org.example;
+
 import java.util.Scanner;
```

Comment:
Remove `import java.util.Scanner;` since is no longer used.

Reply:
done